src/_includes/components/header-nav.njk

```diff
@@ -20,7 +20,7 @@
 <div class="nav-smallScreen">
 	<nav class="starter-nav">
 		<button id="menuToggleButton" aria-expanded="false">
-		  <svg><use xlink:href="#menu-icon" /></svg>&nbsp;Menu
+		  <svg><use xlink:href="#menu-icon" /></svg>&nbsp;<span>Menu</span>
```

Comment:
I think I missed one point in my previous comment. When the menu span is hidden on the Galaxy Fold, the same accessibility issue will occur that ATs won't be able to access this "Menu" text.

A possible fix is to add a `screen-reader-only` span that will be shown on Galaxy Fold and hidden on other screen sizes. Hiding on other screen sizes will prevent the "menu" text being read/accessed twice from 2 different spans.

Reply:
Ready for another review. I am not sure if I did it correctly but I believe this new push is what you are talking about.

Reply:
Awesome fix. Thanks, @TeddTech.